src/main/kotlin/org/openrewrite/kotlin/internal/KotlinParserVisitor.kt

```diff
@@ -3287,7 +3279,7 @@ class KotlinParserVisitor(
                     if (padded.markers.markers.isEmpty()) {
                         padded = padded.withAfter(sourceBefore("->"))
                     } else {
-                        skip("->")
+                        beforeArrow = sourceBefore("->");
```

Comment:
@kunli2 Can you check that the code two lines above adding a right padding is correct?